src/themes/shared/sizes.ts

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable @typescript-eslint/naming-convention */
 export const sizes = {
   xs: '1rem',
   sm: '1.5rem',
   md: '2.25rem',
   lg: '3rem',
   xl: '5rem',
+  '2xl': '6.25rem',
```

Comment:
Is this value already added in figma?

Reply:
I already asked Enrique for adding it, he will do it in figma and I will do it in the figma-tokens.